src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -914,7 +916,20 @@ private static SegueJobService getSegueJobService(final PostgresSqlDb database)
                     "SQL scheduled job that deletes expired reservations for the event booking system",
                     "0 0 7 * * ?", "db_scripts/scheduled/expired-reservations-clean-up.sql");
 
-            segueJobService = new SegueJobService(Arrays.asList(PIISQLJob, cleanUpOldAnonymousUsers, cleanUpExpiredReservations));
+            SegueScheduledJob syncMailjetUsers = new SegueScheduledSyncMailjetUsersJob(
+                    "syncMailjetUsersJob",
+                    "JavaJob",
+                    "Sync users to mailjet",
+                    "0 0 7,19 ? * * *");
```

Comment:
I'm now wondering whether I was wrong when I suggested these times and we should just run it every 6 hours at 12am/6am/12pm/6pm; or even every 4 hours at 12am/4am/8am/12pm/4pm/8pm. Especially in September when we hit the rate limiting a lot, waiting 12 hours before rerunning would mean quite the queue of further changes would back up.

That would be simple to change; if we were to change the Cron string later on, would it update the existing job, or will it be ignored like some other changed properties of jobs are?